test/tarball/verifySemVer.spec.js

```diff
@@ -0,0 +1,74 @@
+// Import Node.js Dependencies
+import path from "node:path";
+import fs from "node:fs/promises";
+import { fileURLToPath } from "node:url";
+import { describe, test } from "node:test";
+import assert from "node:assert";
+
+// Import Internal Dependencies
+import { getSemVerWarning } from "../../src/utils/index.js";
+
+// CONSTANTS
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const FIXTURE_PATH = path.join(__dirname, "..", "fixtures", "verifySemVer");
```

Comment:
```suggestion
const kFixturePath = path.join(__dirname, "..", "fixtures", "verifySemVer");
```
CAPS_CONSTANT is only for exported one